packages/dataviews/src/utils.js

```diff
@@ -49,3 +54,13 @@ export function getPaginationResults( { data, view } ) {
 		},
 	};
 }
+
+export const sanitizeOperators = ( field ) => {
```

Comment:
We can remove the `sanitizeOperators` from view-table and update this implementation to use OPERATORS instead of `OPERATOR_IN, OPERATOR_NOT_IN`.

Reply:
Good catch! I updated.